plugins/multilevel-linear-view/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@jbrowse/plugin-multilevel-linear-view",
+  "version": "1.7.10",
+  "description": "JBrowse 2 multilevel linear view",
+  "keywords": [
+    "jbrowse",
+    "jbrowse2"
+  ],
+  "license": "Apache-2.0",
+  "homepage": "https://jbrowse.org",
+  "bugs": "https://github.com/GMOD/jbrowse-components/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GMOD/jbrowse-components.git",
+    "directory": "plugins/multilevel-linear-view"
+  },
+  "author": "JBrowse Team",
+  "distMain": "dist/index.js",
+  "srcMain": "src/index.tsx",
+  "main": "src/index.tsx",
+  "distModule": "dist/plugin-multilevel-linear-view.esm.js",
+  "module": "",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "start": "tsdx watch --verbose --noClean",
```

Comment:
we don't use tsdx anymore for compilation, can refer to other plugins to see newer plugin build (just babel)